dqc_us_rules/dqc_us_0001.py

```diff
@@ -9,7 +9,7 @@
 from collections import defaultdict
 
 _CODE_NAME = 'DQC.US.0001'
-_RULE_VERSION = '1.1'
+_RULE_VERSION = '2.0'
```

Comment:
Per the semantic versioning thing in the expanded documentation, should this now be `2.0.0`?